webapp/src/pages/RegisterPage/RegisterPage.css

```diff
@@ -1,75 +1,26 @@
-.register {
-  margin: 0;
-  padding: var(--padding-top-bottom) var(--padding-sides);
-  min-height: 100vh;
-  overflow: hidden;
-  background-color: var(--color-background);
-  color: var(--color-light);
-  display: grid;
-  gap: 6rem var(--column-gap);
-  grid-template-columns: repeat(var(--column-number), auto);
-  grid-template-rows: auto 1fr;
-  grid-template-areas:
-    'title title ... ...'
-    'main main main main';
+.register-page {
+  display: contents;
 }
 
-.register-title > *,
-.register-subtitle > * {
-  margin: 0;
-  justify-self: center;
-  max-width: 20rem;
-}
-
-.register-title {
-  grid-area: title;
-}
-
-.register-container {
+.register-form {
   grid-area: main;
   width: 100%;
-  align-self: center;
```

Comment:
nitpick: At 500-700~ px width, the registration form is on the right side, while the login form is in the center.

Possible fix:
<details>

```diff
diff --git a/webapp/src/pages/RegisterPage/RegisterPage.css b/webapp/src/pages/RegisterPage/RegisterPage.css
index 1a3561d..a550984 100644
--- a/webapp/src/pages/RegisterPage/RegisterPage.css
+++ b/webapp/src/pages/RegisterPage/RegisterPage.css
@@ -3,21 +3,21 @@
 }
 
 .register-form {
-  grid-area: main;
   width: 100%;
   display: flex;
   flex-direction: column;
+  align-items: center;
   gap: 0.5rem var(--column-gap);
 }
 
-.register-form .inputs-container > .input {
+.register-form > .input {
   margin-bottom: 0.875rem;
   width: 100%;
   min-width: 15rem;
   max-width: 20rem;
 }
 
-.register-form .inputs-container > .input > label {
+.register-form > .input > label {
   display: inline-block;
   text-align: left;
   width: 100%;
diff --git a/webapp/src/pages/RegisterPage/RegisterPage.tsx b/webapp/src/pages/RegisterPage/RegisterPage.tsx
index 8ed6e1b..362c138 100644
--- a/webapp/src/pages/RegisterPage/RegisterPage.tsx
+++ b/webapp/src/pages/RegisterPage/RegisterPage.tsx
@@ -100,12 +100,12 @@ export default function RegisterPage() {
 
   return (
     <LandingAnimationPageTemplate>
-      <form
-        id="register-form"
-        className="register-form"
-        onSubmit={handleOnSubmit}
-      >
-        <div className="inputs-container">
+      <div className="register-page">
+        <form
+          id="register-form"
+          className="register-form"
+          onSubmit={handleOnSubmit}
+        >
           <Input
             variant={InputVariant.LIGHT}
             label="Username"
@@ -149,8 +149,8 @@ export default function RegisterPage() {
             onChange={handleInputChange}
           />
           <Button variant={ButtonVariant.SUBMIT}>Create new account</Button>
-        </div>
-      </form>
+        </form>
+      </div>
     </LandingAnimationPageTemplate>
   );
 }

```

</details>

Reply:
done, thank you